jetty-core/jetty-quic/jetty-quic-quiche/jetty-quic-quiche-foreign/src/main/java/org/eclipse/jetty/quic/quiche/foreign/ForeignQuicheBinding.java

```diff
@@ -21,27 +21,25 @@
 import org.eclipse.jetty.quic.quiche.QuicheBinding;
 import org.eclipse.jetty.quic.quiche.QuicheConfig;
 import org.eclipse.jetty.quic.quiche.QuicheConnection;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class ForeignQuicheBinding implements QuicheBinding
 {
-    private static final Logger LOG = LoggerFactory.getLogger(ForeignQuicheBinding.class);
-
     @Override
-    public boolean isUsable()
+    public Throwable checkUsable()
     {
         try
         {
-            // Make a Quiche call to confirm.
-            quiche_h.quiche_version();
-            return true;
+            quiche_h.initialize();
+            return null;
+        }
+        catch (ExceptionInInitializerError e)
+        {
+            Throwable cause = e.getCause();
+            return cause != null ? cause : e;
         }
         catch (Throwable x)
         {
-            if (LOG.isDebugEnabled())
-                LOG.debug("java.lang.foreign quiche binding is not usable", x);
-            return false;
+            return x;
```

Comment:
Store the exception to be used in `toString()`.

Reply:
Done.